parser.jison

```diff
@@ -39,6 +39,7 @@
 
 (calc)                                 return 'NESTED_CALC';
 (var\([^\)]*\))                        return 'CSS_VAR';
+([a-z]+\([a-z0-9-]+\))                 return 'UNKNOW_FUNCTION'
```

Comment:
The regex doesn't seem very resilient to me. Furthermore it might be worth normalising this and the CSS variable stuff to a single thing. \cc @MoOx what do you think?